stable/metricbeat/templates/secret.yaml

```diff
@@ -72,8 +72,8 @@ kind: Secret
 metadata:
   name: {{ $secret.name }}
   labels:
-    app: {{ template "metricbeat.name" . }}
```

Comment:
Would you mind trying `{{ template "metricbeat.name" $ }}` instead? The `$` always point to the "root" template data which isn't mutated by `range`s.

Reply:
Oh..Thank you for your good feedback :)

Reply:
@mumoshu